CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 5.0.0
+
+- **Breaking change**: Requires Dart 3.0 or above
```

Comment:
Is Dart 3 strictly required?  This is a problem for us as we haven't yet completed the migration to Dart 3 and are still on Dart 2.19.